db-scripts/src/main/resources/cgds.sql

```diff
@@ -111,6 +111,20 @@ CREATE TABLE `type_of_cancer` (
   PRIMARY KEY (`TYPE_OF_CANCER_ID`)
 );
 
```

Comment:
I see 2 references to dropping table `reference_genome_gene`

- [L85](https://github.com/cBioPortal/cbioportal/pull/5891/files?file-filters%5B%5D=.sql#diff-0146a84b4bc5ff8d2ac00911a327d25fR85)
- [L100](https://github.com/cBioPortal/cbioportal/pull/5891/files?file-filters%5B%5D=.sql#diff-0146a84b4bc5ff8d2ac00911a327d25fR100)

Reply:
good catch, removed line 100.